stairway/src/main/java/bio/terra/stairway/impl/StairwayImpl.java

```diff
@@ -484,10 +497,14 @@ public void deleteFlight(String flightId, boolean forceDelete)
    * @throws FlightNotFoundException flight id does not exist
    * @throws FlightWaitTimedOutException if interrupted or polling interval expired
    * @throws InterruptedException on shutdown while waiting for flight completion
+   * @deprecated
    */
   public FlightState waitForFlight(String flightId, Integer pollSeconds, Integer pollCycles)
```

Comment:
This is OOS, but I don't love how this takes in `pollSeconds` rather than a `Duration` to poll for.  Tests that use this method end up taking a lot longer to run on account of the default 10s sleeps.  But, hopefully this is a moot point given the deprecation.